settings.local.generic.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Local Backdrop CMS configuration file.
+ */
+
+/**
+ * Simple database settings:
+ *
+ * Most sites can configure their database by entering the connection string
+ * below. If using master/slave or multiple connections, see the advanced
+ * database settings.
+ */
+require_once __DIR__ . '/civicrm_table_prefixes.php';
+$databases = array(
+  'default' => array (
+    'default' =>
+    array (
+      'driver' => 'mysql',
+      'database' => 'backdropcmsorg',
+      'username' => '',
+      'password' => '',
+      'host' => '127.0.0.1',
+      'prefix' => $civicrm_table_prefixes,
```

Comment:
Do we need civi prefixes for the default/backdrop database? I think we only need these on the civi database? 

I don't think I have this for my local setup...  checking... no, I'm still using 
`$database = 'mysql://username:password]@127.0.0.1/backdropcms';` with no prefixes at all.

Reply:
We only need it for with-CiviCRM, can use the simple version for without. But there's no harm in having it there (that I know of) since the prefix file is in the repo. I'm guessing you'd like to tell people to use the simpler version if they're not running CiviCRM, right?

Reply:
I'll use both.